src/content/peerconnection/bandwidth/js/main.js

```diff
@@ -112,8 +112,10 @@ function gotDescription2(desc) {
   pc2.setLocalDescription(desc).then(
     function() {
       trace('Answer from pc2 \n' + desc.sdp);
-      desc.sdp = updateBandwidthRestriction(desc.sdp, '500');
-      pc1.setRemoteDescription(desc).then(
+      pc1.setRemoteDescription({
+        type: desc.type,
+        sdp: updateBandwidthRestriction(desc.sdp, '500')
```

Comment:
I'm pretty sure I have some laying around which allows to use the bandwidth drop down for the initial OA. I'll merge and submit that once this is merged.